forte_wrapper/vader/sentiment_analysis.py

```diff
@@ -51,24 +50,37 @@ def __init__(self):
 
     def initialize(self, resources: Resources, configs: Config):
         # pylint: disable=unused-argument
+        super().initialize(resources, configs)
         self.sentence_component = configs.get('sentence_component')
 
     def _process(self, input_pack: DataPack):
-        for sentence in input_pack.get(entry_type=Sentence,
-                                       components=self.sentence_component):
-            scores = self.analyzer.polarity_scores(sentence.text)
-            sentence.sentiment = scores
+        for entry_specified in input_pack.get(
+                entry_type=self.configs.entry_type,
+                components=self.sentence_component):
+            scores = self.analyzer.polarity_scores(entry_specified.text)
+            setattr(entry_specified, self.configs.attribute_name, scores)
+
 
     @classmethod
     def default_configs(cls):
         r"""This defines a basic config structure for VaderSentimentProcessor.
-
-        sentence_component (str): If not None, the processor will process
+        Returns:
+            dictionary with the default config for this processor.
+        Following are the keys for this dictionary:
```

Comment:
I am not sure if the indentation here work. You can discuss with Jenny to see what's the correct way to do this.

Reply:
Updated and Checked through the html, it seems perfect now.